### DISCUSSION TITLE
可以考虑加给用户限速的功能吗

### DISCUSSION THREAD

**Discussion by rheatin** [AlistGo/alist#1949] [Ideas]
多用户情景下，看电影的女朋友，影响到我办公了- -

**Comment by youtehub** [comment]
服务多开不就行了嘛，给你女朋友单独部署一个alist服务器，也就是说，一个alist服务器用来工作，另外一个alist服务器用来生活娱乐

**Reply by rheatin** [reply]
办法肯定是有的嘛，但是这个Alist既然有多用户，肯定会有人做成网盘形式的吧，我觉得给用户加一个QOS功能应该也是不错的选择的吧~

**Reply by haorwen** [reply]
eee，如果不开代理模式大多是直接走网盘的服务器，这个估计不好限制，开了代理当我没说..

**Reply by zhendery** [reply]
服务多开可以自动分配带宽吗？主要是有人在下载东西时导致别人视频会卡住。
